### PR TITLE
Minor doc updates

### PR DESCRIPTION
**Pull request by bkorycki**

### CODE REVIEW

File: docs/tutorial_tests.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ Let's say we want to create a Test where we send a bunch of questions to the SUT
 To run these questions as a Test in NewHELM, we need (for now) to create a class for our Test. Let's call our Test `DemoSimpleQATest`.
 Since this fits as a [Prompt Response Test](prompt_response_tests.md) we can have it inherit from `BasePromptResponseTest`. We now have several abstract methods we need to define.
 
-The first phase in a PromptResponseTest is making the `TestItem`s. We want these to be our questions from `an_example.jsonl`. NewHELM uses [DependencyHelper](../newhelm/dependency_helper.py) to ensure good hygiene of data dependencies (e.g. versioning). So we first need to tell NewHELM that we have a dependency on that file by listing it in `get_dependencies`:
+### 1. Make the test items
+The first phase in a PromptResponseTest is making the `TestItem`s. We want these to be our questions from `an_example.jsonl`. NewHELM uses [DependencyHelper](../newhelm/dependency_helper.py) to ensure good hygiene of data dependencies (e.g. versioning). So we first need to tell NewHELM that we have a dependency on that file by listing it in **`get_dependencies`**:
```
Comment (brianwgoldman) [inline docs/tutorial_tests.md:17]: Here and throughout, why change from `code` to **`strong code`**?

File: docs/tutorial_tests.md [hunk 2]
```diff
@@ -26,22 +27,26 @@ def get_dependencies(self):
 
 We'll explore more of what DependencyHelper can do in later demos. Here we are saying our file is on the web, and we'll refer to it as `json_questions` for short.
 
-The `make_test_items` method is where we convert our `an_example.jsonl` file into TestItems. `DependencyHelper` manages downloading the file and provides us with a path to it:
+The **`make_test_items`** method is where we convert our `an_example.jsonl` file into TestItems. `DependencyHelper` manages downloading the file and provides us with a path to it:
```
Comment (brianwgoldman) [inline docs/tutorial_tests.md:30]: DependencyHelper is also code, but not bolded. Probably good to be consistent.

File: docs/tutorial_tests.md [hunk 3]
```diff
@@ -26,22 +27,26 @@ def get_dependencies(self):
...
+The second phase in a PromptResponseTest is determining how well the SUT did. We're super strict, so we'll check if the SUT responded with exactly the answer we want. In **`measure_quality`**, we get back each TestItem, but now with data about what the SUT did:
 
 ```py
 interaction.response.completions[0].text == interaction.prompt.context
 ```
 
-We can then use one of the provided [aggregation functions](../newhelm/aggregations.py) to determine how often the SUT responded correctly.
+### 3. Aggregate the measurements
 
+We can then use one of the provided [aggregation functions](../newhelm/aggregations.py) to determine how often the SUT responded correctly in **`aggregate_measurements`**.
```
Comment (brianwgoldman) [inline docs/tutorial_tests.md:47]: While you are here, can you call out that the user could write their own if none of the existing ones worked?

**Review by brianwgoldman — APPROVED** [review]

**Review by bollacker — APPROVED** [review]
LGTM

File: docs/tutorial_tests.md (hunk 1 above)
Comment (bkorycki) [reply docs/tutorial_tests.md:17]: I was just bolding the Test method names that correspond to each step. I can undo that if you think the inconsistency is awkward.

Comment (brianwgoldman) [reply docs/tutorial_tests.md:17]: Ah, yeah, at least to me that wasn't clear. I prefer not bolding, but its not a strong preference.

File: docs/tutorial_tests.md (hunk 3 above)
Comment (bkorycki) [reply docs/tutorial_tests.md]: Does my edit convey what you had in mind? Or did you mean the user can write their own function outside of aggregate_measurements, like in aggregations.py or something?

File: docs/tutorial_tests.md (hunk 1 above)
Comment (bkorycki) [reply docs/tutorial_tests.md]: I'll undo it in that case

**Pull request merged**